libs/superagent/app/vectorstores/test_upserted.py

```diff
@@ -0,0 +1,24 @@
+import requests
+import json
+
+KEYSPACE_NAME = "recommender_demo"
+COLLECTION_NAME = "superagent_vector_json"
+ASTRA_DB_ID = "8b400d8c-9cd5-436c-9e3a-59c1223fc993"
```

Comment:
can you remove the creds from here, we shouldn't push these to github

Reply:
Fixed it